tests/integration/test_session.py

```diff
@@ -353,3 +367,96 @@ def generate_random2(row) -> int:
 
     assert isinstance(sum_random, int), "Result is not of type int"
     connect_session.stop()
+
+
+def test_sql_functions(connect_session):
+    """Test basic SQL functions like col(), sum(), count(), etc."""
+    # Create a test DataFrame
+    df = connect_session.createDataFrame(
+        [(1, "Alice", 100), (2, "Bob", 200), (3, "Charlie", 150)],
+        ["id", "name", "amount"],
+    )
+
+    # Test col() function
+    result_col = df.select(F.col("name")).collect()
```

Comment:
Will it fail if we revert https://github.com/GoogleCloudDataproc/dataproc-spark-connect-python/pull/112?